AutomatedTesting/Gem/PythonTests/Atom/tests/MaterialEditor_Atom_PeriodicTests.py

```diff
@@ -0,0 +1,409 @@
+        starting_first_material_document_color = material_editor_utils.get_property(
+            first_material_document_id, base_color_property_name)
+        first_material_document_color = azlmbr.math.Color(0.0156, 0.0196, 0.0862, 1.0)
+        material_editor_utils.set_property(
+            first_material_document_id, base_color_property_name, first_material_document_color)
+        Report.result(
+            Tests.changed_first_material_document_color,
+            material_editor_utils.get_property(
+                first_material_document_id, base_color_property_name) == first_material_document_color)
+
+        # 16. Save the first_material_document as a new material document file.
+        first_material_document_saved = atom_tools_utils.save_document(first_material_document_id)
+        Report.result(
+            Tests.saved_first_material_document,
+            first_material_document_saved is True)
+
+        # 17. Open a second material document referred to as "second_material_document".
+        second_material_document_id = atom_tools_utils.open_document(second_material_document)
+        Report.result(
+            Tests.opened_second_material_document,
+            atom_tools_utils.is_document_open(second_material_document_id) is True)
+
+        # 18. Change the baseColor.color property value of the second_material_document to 0.4156, 0.0196, 0.6862, 1.0.
+        starting_second_material_document_color = material_editor_utils.get_property(
+            second_material_document_id, base_color_property_name)
+        second_material_document_color = azlmbr.math.Color(0.4156, 0.0196, 0.6862, 1.0)
+        material_editor_utils.set_property(
+            second_material_document_id, base_color_property_name, second_material_document_color)
+        Report.result(
+            Tests.changed_second_material_document_color,
+            material_editor_utils.get_property(
+                second_material_document_id, base_color_property_name) == second_material_document_color)
+
+        # 19. Save the second_material_document as a new material document file.
+        second_material_document_saved = atom_tools_utils.save_document(second_material_document_id)
+        Report.result(
+            Tests.saved_second_material_document,
+            second_material_document_saved is True)
+
+        # 20. Close the first_material_document and second_material_document.
+        first_material_document_closed = atom_tools_utils.close_document(first_material_document_id)
+        Report.result(
+            Tests.closed_first_material_document,
+            first_material_document_closed is True)
+        second_material_document_closed = atom_tools_utils.close_document(second_material_document_id)
+        Report.result(
+            Tests.closed_second_material_document,
+            second_material_document_closed is True)
```

Comment:
should we check `is_document_open` is false for these make sure they are not open.

Reply:
sure we can add extra sanity